mmedit/models/restorers/dic.py

```diff
@@ -0,0 +1,143 @@
+from collections import OrderedDict
+
+from mmedit.models.common import ImgNormalize
+from ..builder import build_backbone, build_loss
+from ..registry import MODELS
+from .basic_restorer import BasicRestorer
+
+
+@MODELS.register_module()
+class DIC(BasicRestorer):
+    """DIC model for Face Super-Resolution.
+
+    Paper: Deep Face Super-Resolution with Iterative Collaboration between
+        Attentive Recovery and Landmark Estimation.
+
+    Args:
+        generator (dict): Config for the generator.
+        pixel_loss (dict): Config for the pixel loss.
+        align_loss (dict): Config for thr align loss.
+        train_cfg (dict): Config for train. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def __init__(self,
+                 generator,
+                 pixel_loss,
+                 align_loss,
+                 train_cfg=None,
+                 test_cfg=None,
+                 pretrained=None):
+        super(BasicRestorer, self).__init__()
+
+        self.train_cfg = train_cfg
+        self.test_cfg = test_cfg
+
+        # model
+        self.generator = build_backbone(generator)
+        self.img_denormalize = ImgNormalize(
```

Comment:
Where is it used?

Reply:
Out of curiosity, where is the normalize corresponds to this denormalize?

Reply:
The normalize is in pipeline (called in the config file, similar to other method of this repo).

Reply:
How about the first question

Reply:
Q: Where is it used?
A: Deleted by mistake in the previous version (f0b85d2).

This problem has been fixed.

Reply:
Can this de-normalization be configurable? Just asking... We can leave the better normalize-denormalize story to future work. Currently they are quite ad-hoc.

Reply:
post-pipeline?

Reply:
Maybe...